### ISSUE TITLE
Option to see how long the task was waiting in the queue before being handled

### ISSUE THREAD

**Issue by matancarmeli7** [danihodovic/celery-exporter#316]
Is there an option to see how long the task was waiting in the queue before being handled with the exporter metrics?

**Comment by danihodovic** [comment]
I don't think so, but if celery_queue_length is larger than 0 it's guaranteed to be put on hold.

**Comment by matancarmeli7** [comment]
Thanks for the quick answer :), there is a plan to add it in the future? 
because I see [here], that they have done [here](https://github.com/OvalMoney/celery-exporter?tab=readme-ov-file#command-options) something that can be relevant to that
![image](https://github.com/user-attachments/assets/fc1cfcec-432c-4e04-bbe8-f98a0ff2eb6d)

**Comment by danihodovic** [comment]
Feel free to open a pull-request :-)

**Comment by azhar1038** [comment]
I need this too!
Are you working on this @matancarmeli7 ?
